apply-templates.sh

```diff
@@ -41,6 +41,9 @@ for version; do
 			gawk -f "$jqt" "Dockerfile-$variant.template"
 		} > "$version/$variant/Dockerfile"
 
+		conf='rabbitmq.conf'
+		cp -a "$conf" "$version/$variant/rabbitmq.conf"
```

Comment:
No need for the `conf` variable here (we only have a variable for `entrypoint` because it changes based on version).

Reply:
done